includes/class-scd-ext-lesson-frontend.php

```diff
@@ -47,13 +47,16 @@ class Scd_Ext_Lesson_Frontend {
 	 * Constructor function
 	 */
 	public function __construct() {
-		// Set a formatted  message shown to user when the content has not yet dripped
-		$default_message = 'This lesson will become available on [date].';
-		$settings_field =  'scd_drip_message';
-		$this->message_format = Sensei_Content_Drip()->utils->check_for_translation($default_message, $settings_field );
+		// Set a formatted  message shown to user when the content has not yet dripped.
+		$default_message      = 'This lesson will become available on [date].';
+		$settings_field       = 'scd_drip_message';
+		$this->message_format = Sensei_Content_Drip()->utils->check_for_translation( $default_message, $settings_field );
 
-		// Hook int all post of type lesson to determine if they should be
-		add_filter('the_posts', array( $this, 'lesson_content_drip_filter' ), 1 );
+		// Hook int all post of type lesson to determine if they should be.
+		add_filter( 'the_posts', array( $this, 'lesson_content_drip_filter' ), 1 );
+
+		// Add action to not display comments if the lesson is blocked.
+		add_action( 'sensei_pagination', array( $this, 'hide_comments' ), 80 );
```

Comment:
You likely don't need to specify a priority here, as the default is 10 which will beat the original one that uses a priority of 90.